src/components/MotifLanding.jsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
```

Comment:
Looks like you have expanded `@databyss-org/ui/modules/Landing/Landing` into the `Landing/MotifLanding` component. Instead, make the changes you need in `@databyss/ui` to implement sticky headers